dexmaker/src/test/java/com/google/dexmaker/DexMakerTest.java

```diff
@@ -66,6 +67,15 @@
     private void reset() {
         dexMaker = new DexMaker();
         dexMaker.declare(GENERATED, "Generated.java", PUBLIC, TypeId.OBJECT);
+        clearDataDirectory();
+    }
+
+    private void clearDataDirectory() {
```

Comment:
The nesting directory structure applies to ProxyBuilder, not DexMaker. It's ProxyBuilder.dexCache() that creates the nesting structure. For the tests in DexMakerTest, the dex files are being written into the data directory with no nesting.

Without getting into details, it's actually difficult to perform a recursive delete in the data directory when running vogar because vogar installs the apks required to run the unit tests inside this directory structure.